### PR TITLE
OpenAPIEnums feature is beta in 1.24

### PR DESCRIPTION
**Pull request by tengqm**
KEP: kubernetes/enhancements#2887
Upstream: kubernetes/kubernetes#108898

### CODE REVIEW

**Comment by jiahuif** [comment]
I thought this was auto updated after code freeze. Am I correct?

**Comment by tengqm** [comment]
@jiahuif Unfortunately, that is not true. There is no such automation in place and I don't see it will happen any time soon because the feature state is coded in Go source instead of a configuration file. The features are managed by a dozen of SIGs collectively, and there is more than one source of truth. We have to manually maintain this and carry on.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -155,7 +155,8 @@ different Kubernetes components.
 | `NetworkPolicyEndPort` | `true` | Beta | 1.22 |  |
 | `NodeSwap` | `false` | Alpha | 1.22 | |
 | `NodeOutOfServiceVolumeDetach` | `false` | Alpha | 1.24 | |
-| `OpenAPIEnums` | `false` | Alpha | 1.23 | |
+| `OpenAPIEnums` | `false` | Alpha | 1.23 | 1.23 |
+| `OpenAPIEnums` | `false` | Beta | 1.24 | |
```
Comment (apelisse) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:159]: I think it's enabled by default, if I'm correct. @jiahuif ?

Comment (jiahuif) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:159]: Correct. That's a problem. Sending a fix.

**Pull request merged**